intro.stories.mdx

```diff
@@ -57,28 +57,22 @@ In this [playground](https://components.novatics.com.br/?path=/docs/currency-tex
 yarn install
 ```
 
-2. Use nx generators to create the component. Visit the [Nx Documentation](https://nx.dev) to learn more.
+2. Use nx generators to create the component and the storybook documentation. Visit the [Nx Documentation](https://nx.dev) to learn more.
 
 ```shell
-yarn generate:component <name-of-component>
+yarn component:generate 
```

Comment:
Is the `name-of-component` no longer required?

Reply:
never was. if the name is not passed, nx opens a prompt